### PR TITLE
Adding IIWA package to documentation index

### PR DESCRIPTION
**Pull request by khansari**

### CODE REVIEW

File: indigo/distribution.yaml [hunk 1]
```diff
@@ -8227,5 +8227,20 @@ repositories:
...
       version: indigo
     status: developed
+  IIWA:
+    doc:
+      type: git
+      url: https://khansari@bitbucket.org/khansari/iiwa.git
+      version: hydro-devel
+    release:
+      tags:
+        release: release/hydro/{package}/{version}
```
Comment (dirk-thomas) [inline indigo/distribution.yaml]: This is certainly the wrong release tag: it contains *hydro*

File: hydro/distribution.yaml [hunk 2]
```diff
@@ -9270,5 +9270,20 @@ repositories:
       url: https://github.com/stonier/zeroconf_msgs.git
       version: hydro-devel
     status: developed
+  IIWA:
+    doc:
+      type: git
+      url: https://khansari@bitbucket.org/khansari/iiwa.git
```
Comment (dirk-thomas) [inline hydro/distribution.yaml]: The URL should no contain your username with the @ sign.

File: hydro/distribution.yaml [hunk 3]
```diff
@@ -9270,5 +9270,20 @@ repositories:
...
+      version: hydro-devel
```
Comment (dirk-thomas) [inline hydro/distribution.yaml]: Your repository does not have a *hydro-devel* branch - only *master*.

File: hydro/distribution.yaml [hunk 4]
```diff
@@ -9270,5 +9270,20 @@ repositories:
...
+    release:
+      tags:
+        release: release/hydro/{package}/{version}
+      url: https://bitbucket.org/khansari/iiwa/wiki/Home
```
Comment (dirk-thomas) [inline hydro/distribution.yaml]: This url must point to a git-buildpackage release repository. Please read about `bloom` and how to release a ROS package.

Comment (wjwwood) [inline hydro/distribution.yaml]: http://wiki.ros.org/bloom

**Pull request closed without merging**